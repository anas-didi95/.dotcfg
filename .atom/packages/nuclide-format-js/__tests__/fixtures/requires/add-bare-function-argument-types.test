declare function fancy(cb: Some => Other): void;
