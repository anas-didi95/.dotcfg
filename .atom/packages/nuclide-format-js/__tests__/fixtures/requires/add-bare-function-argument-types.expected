import type Other from 'Other';
import type Some from 'Some';

declare function fancy(cb: Some => Other): void;
